doc/source/ray-contribute/development.rst

```diff
@@ -14,103 +14,58 @@ For a majority of Ray users, installing Ray via the latest wheels or pip package
 .. contents::
   :local:
 
-Clone the repository
+Fork the Ray repository
 --------------------
```

Comment:
This line is breaking the RTD build. Extend the `-` to match the line length of the title above.